### PR TITLE
New consulting page - .NET 8 Migration

### LINKED ISSUE #1561: 📄 New consulting page - .NET 8 Migration
### 1. People in the loop:

@samsmithssw @UlyssesMaclaren @william-liebenberg 

### 2. Page template to copy:

Consulting

### 3. CRM Record (only for consulting service pages):

https://ssw.crm6.dynamics.com/main.aspx?appid=05daa2a9-8768-446e-b9d3-580c8a6f9b8b&pagetype=entityrecord&etn=ssw_consultingservice&id=4c2d8c43-3e6e-ee11-9ae7-6045bd3df3b1

### 4. URI:

net-8-migration

### 5. Title:

Enterprise Development and Consulting for .NET 8 Migration

### 6. Meta description:

Enterprise Development and Consulting for .NET 8 Migration. Specialized in LTS migration, Microsoft-certified consultants, cloud-native development. Phased migration approach. Futureproof your applications.

### 7. Related URLs:

none

### 8. Content:

https://sswcom.sharepoint.com/:w:/s/Chatbot/Eas_zmwXPl9Jt_lLs1d5J_sB_6EbMquxqAt7G1Hzpys7kg?e=jfeeyF

### PR DESCRIPTION
**Pull request by sethdaily**
@samsmithssw @UlyssesMaclaren @camillars @william-liebenberg 
As per email subject: .NET 8 Migration Consulting Page

- Fixed #1561 

- [x] If adding a new page, I have followed the [SEO checklist](https://www.ssw.com.au/rules/seo-checklist/)

### CODE REVIEW

File: content/consulting/index/index.json [hunk 1]
```diff
@@ -90,13 +90,13 @@
...
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
             }
           ]
         },
         {
           "title": "Jamstack (Static Sites with Gatsby/Next.js)",
-          "description": "Jamstack is a modern web development architecture based on client-side JavaScript, reusable APIs and prebuilt Markup. It's used on SSW Rules and SSW People.",
+          "description": "Jamstack is a modern  development architecture based on client-side JavaScript, reusable APIs and prebuilt Markup. It's used on SSW Rules and SSW People.",
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
          "description": "Jamstack is a modern web development architecture based on client-side JavaScript, reusable APIs and prebuilt Markup. It's used on SSW Rules and SSW People.",
```

File: content/consulting/index/index.json [hunk 2]
```diff
@@ -90,7 +90,7 @@
               "tag": "content/consulting/tag/mobile-development.json"
             },
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/index/index.json [hunk 3]
```diff
@@ -27,12 +27,12 @@
         },
         {
           "title": "Angular / SPA",
-          "description": "Build sexy web apps with this a TypeScript-based web application platform, open-sourced and led by Google!",
+          "description": "Build sexy  apps with this a TypeScript-based  application platform, open-sourced and led by Google!",
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
          "description": "Build sexy web apps with this a TypeScript-based web application platform, open-sourced and led by Google!",
```

File: content/consulting/index/index.json [hunk 4]
```diff
@@ -32,7 +32,7 @@
           "page": "content/consulting/angular.mdx",
           "tags": [
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/index/index.json [hunk 5]
```diff
@@ -43,7 +43,7 @@
           "page": "content/consulting/react.mdx",
           "tags": [
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/index/index.json [hunk 6]
```diff
@@ -54,7 +54,7 @@
           "page": "content/consulting/blazor.mdx",
           "tags": [
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/index/index.json [hunk 7]
```diff
@@ -65,18 +65,18 @@
           "page": "content/consulting/nextjs.mdx",
           "tags": [
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/index/index.json [hunk 8]
```diff
@@ -71,12 +71,12 @@
         },
         {
           "title": "Vue.js",
-          "description": "Vue.js is a progressive JavaScript framework to build web interfaces and applications.",
+          "description": "Vue.js is a progressive JavaScript framework to build  interfaces and applications.",
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
          "description": "Vue.js is a progressive JavaScript framework to build web interfaces and applications.",
```

File: content/consulting/index/index.json [hunk 9]
```diff
@@ -76,7 +76,7 @@
           "page": "content/consulting/vue.mdx",
           "tags": [
             {
-              "tag": "content/consulting/tag/web-development.json"
+              "tag": "content/consulting/tag/-development.json"
```
Comment (sethdaily) [inline content/consulting/index/index.json]:
```suggestion
              "tag": "content/consulting/tag/web-development.json"
```

File: content/consulting/net-8.mdx [hunk 10]
```diff
@@ -0,0 +1,79 @@
...
+
+.NET 8 provides robust enterprise functionality and cloud-native development all within a stable ecosystem backed by Microsoft.
+
+For your team, .NET 8 is the ultimate dev experience, updated with state-of-the-art features like C# 12, Native AOT (Ahead of Time Compilation), and source code generation.
+
+# Why SSW?
+
+Migration is complex, risky, and costly if not done correctly. SSW specializes in migrating applications, and our team of Microsoft certified consultants have a deep understanding of the .NET platform.
+
+We have an established history of helping clients migrate intricate systems to the newest version of .NET, and we have documented our learnings on our [Rules to Better .NET 8 Migration.](https://www.ssw.com.au/rules/rules-to-better-net8-migrations/)
```
Comment (Harry-Ross) [inline content/consulting/net-8.mdx:77]: You can add this as the rule link as well if you want in the "rule" section of benefits:

image
**Figure: The additional rules link added on /consulting/nextjs**

image
**Figure: The fields added in nextjs.mdx**

**Review by Harry-Ross — APPROVED** [review]

**Pull request merged**